examples/oriented_images.html

```diff
@@ -0,0 +1,160 @@
+                border: 2px solid black;
+                border-radius: 5px;
+            }
+        </style>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <script src="js/GUI/dat.gui/dat.gui.min.js"></script>
+        <script src="js/OrientedImageHelper.js"></script>
+    </head>
+    <body>
+        <div id="viewerDiv">
+            <p id="info">
+                Picture taken by Walter Mittelholzer in 1919, metadata from 
+                <a href="https://smapshot.heig-vd.ch/map/?imageId=2873" target="_blank"> smapshot.heig-vd.ch </a>
+            </p>
+        </div>
+        <script src="js/GUI/GuiTools.js"></script>
+        <script src="../dist/itowns.js"></script>
+        <script src="../dist/debug.js"></script>
+        <script type="text/javascript">
+            // Define initial camera position.
+            // Note that camera initial position will be overwritten by picture position later
+            var dummyPositionOnGlobe = { longitude: 0, latitude: 0, altitude: 10000000 };
+
+            // `viewerDiv` will contain iTowns' rendering area (`<canvas>`)
+            var viewerDiv = document.getElementById('viewerDiv');
+
+            // Instanciate iTowns GlobeView*
+            var globeView = new itowns.GlobeView(viewerDiv, dummyPositionOnGlobe, {
+                sseSubdivisionThreshold: 6,
+                noControls: true,
+            });
+
+            var pictureInfos;
+            var coord;
+            var rotationMatrix;
+            // eslint-disable-next-line no-unused-vars
+            var plane;
+            var camera;
+
+            function addLayerCb(layer) {
+                return globeView.addLayer(layer);
+            }
+
+            // Add one imagery layer to the scene
+            // This layer is defined in a json file but it could be defined as a plain js
+            // object. See Layer* for more info.
+            itowns.Fetcher.json('./layers/JSONLayers/Ortho.json').then(addLayerCb);
+
+            // Add two elevation layers.
+            // These will deform iTowns globe geometry to represent terrain elevation.
+            itowns.Fetcher.json('./layers/JSONLayers/WORLD_DTM.json').then(addLayerCb);
+            itowns.Fetcher.json('./layers/JSONLayers/IGN_MNT_HIGHRES.json').then(addLayerCb);
+
+            pictureInfos = {
+                panoramic: {
+                    latitude: 45.9228208442959,
+                    height: 4680.55588294683,
+                    longitude: 6.83256920100156,
+                    azimuth: 526.3900321920207,
+                    roll: 2.1876227239518276,
+                    tilt: -11.668910605126001,
+                    image: 'http://www.itowns-project.org/itowns-sample-data/images/MontBlanc/MontBlanc.jpg',
+                },
+                camera: {
+                    // size of the picture in pixel
+                    size: [6490, 4408],
```

Comment:
You should remove this field completely because:
1. it's bogus in your example because it doesn't match the image resolution
2. you can deduce it from the image, so there's no need to ask the user
3. it's not a `camera` parameter (it's an image parameter)

And you should keep the image ratio when reiszing the image (even if threejs complains about non-power-of-2 texture).

Reply:
I am with @gliegard here : This `size` here is not the size of the image that is to be rendered but the size of the image that was used for the photogrammetric calibration. The beauty of projective texturing and uv mapping is that you do not care about pixel coordinates but about uv coordinates that are normalized to 0-1, so you can use any resized version of the same image with the same photogrametric calibration (even anisotropic ones that change the aspect ratio with different scaling in x and y). This was used in itowns-legacy where the same calibration was used for multiple resized versions of the same image.